test/api.js

```diff
@@ -478,6 +498,75 @@ describe('/avatars', function() {
   });
 });
 
+describe('/display_name', function() {
+  var tok = token();
+
+  describe('GET', function() {
+    it('should return a displayName', function() {
+      mock.token({
+        user: USERID,
+        email: 'user@example.domain',
+        scope: ['profile:display_name']
+      });
+
+      return db.setDisplayName(USERID, 'Spock')
+      .then(function() {
+        return Server.api.get({
+          url: '/display_name',
+          headers: {
+            authorization: 'Bearer ' + tok
+          }
+        }).then(function(res) {
+          assert.equal(res.statusCode, 200);
+          assert.equal(JSON.parse(res.payload).displayName, 'Spock');
+        });
+      });
+    });
+
```

Comment:
I used `{}` to be consistent with `GET /avatar` when there's no avatar, but 404 would be more RESTy. We could break consistency here and use 404. Eventually we could update avatar and clients to handle 404 as well, or just leave it inconsistent. Or stay with an empty response and 200 status and be less RESTy. I'm not really opinionated on it, but maybe using a 404 is fine and we can live with the inconsistency.